target/karate-reports/karate-summary-json.txt

```diff
@@ -0,0 +1 @@
+{"efficiency":0.3402061855670103,"totalTime":1188.0,"threads":1,"resultDate":"2023-07-31 02:29:28 pm","env":null,"version":"1.3.1","scenariosfailed":0,"featureSummary":[{"failedCount":0,"packageQualifiedName":"src.test.karate.features.post-register.post-register","relativePath":"src\/test\/karate\/features\/post-register\/post-register.feature","scenarioCount":1,"name":"As an interested user, I would like to successfully register for an account","description":"","durationMillis":1187.689751,"passedCount":1,"failed":false}],"featuresPassed":1,"featuresFailed":0,"featuresSkipped":0,"scenariosPassed":1,"elapsedTime":3492.0}
```

Comment:
Can we show the new report after you run the test? In practical we shouldn't put the generated report into code repository. Same to all files in the `target` directory.